src/simplewallet/simplewallet.cpp

```diff
@@ -1492,61 +1598,43 @@ bool simple_wallet::sign_multisig_main(const std::vector<std::string> &args, boo
   SCOPED_WALLET_UNLOCK_ON_BAD_PASSWORD(return false;);
 
   std::string filename = args[0];
-  std::vector<crypto::hash> txids;
+  std::vector<std::string> txids;
   uint32_t signers = 0;
```

Comment:
`signers` will be needed for an info message below, but it seems your code, unlike the old code, does not set it any longer?